src/App.js

```diff
@@ -1,10 +1,26 @@
+/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
-import CalculatorUI from './Components/calculatorUI';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+// eslint-disable-next-line import/extensions
+import CalculatorUI from './Components/CalculatorUI.js';
+import Quotes from './Components/Quotes';
+import Home from './Components/Home';
 
-class App extends React.PureComponent {
+class App extends React.Component {
   render() {
     return (
-      <CalculatorUI />
+      <Router>
+        <Routes>
+          <Route path="/Quotes" element={<Quotes />} />
+          <Route path="/CalculatorUI" element={<CalculatorUI />} />
```

Comment:
- [ ] Following the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md). There should be a text on the left and the calculator UI on the right in the calculator page,

Reply:
done